docs/sources/tempo/configuration/_index.md

```diff
@@ -1312,6 +1323,17 @@ overrides:
     #  in the front-end configuration is used.
     [max_search_duration: <duration> | default = 0s]
 
+    # *vParquet3-only configuration*
+    # Configures attributes to be stored as dedicated columns in the parquet file, rather than in the
+    # generic attribute key-value list. This allows for more efficient searching of these attributes.
+    # Up to 10 span attributes and 10 resource attributes can be configured as dedicated columns.
```

Comment:
```suggestion
    # Configures attributes to be stored as dedicated columns in the parquet file, rather than in the
    # generic attribute key-value list. This allows for more efficient searching of these attributes.
    # Up to 10 span attributes and 10 resource attributes can be configured as dedicated columns.
    # Requires vParquet3
```